src/ngraph/pass/algebraic_simplification.cpp

```diff
@@ -543,6 +544,83 @@ static bool simplify_reduction(shared_ptr<Node> n)
     return true;
 }
 
+static bool replace_transpose_with_reshape(shared_ptr<Node> n)
+{
+    bool rc = false;
+    auto transpose = as_type_ptr<op::Transpose>(n);
```

Comment:
use explicit op version, v1::Transpose, helps clarity , and avoid future conflicts if any